### PR TITLE
NGIX Add Always header option

### PR DESCRIPTION
**Pull request by inetbiz**
Additional background at http://nginx.org/en/docs/http/ngx_http_headers_module.html for the **always** parameter.

### CODE REVIEW

File: pages/hsts.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Zooming out even further: it's technically possible to preload HSTS for an entir
 On **nginx**, you would apply an `add_header` command to the appropriate virtual host configuration. This website, `https.cio.gov`, is hosted on nginx, and uses [this batch of HTTPS rules](https://github.com/fisma-ready/nginx/blob/master/ssl/ssl.rules) to set this header:
 
 ```
-add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
+add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload; always';
```
Comment (alex) [inline pages/hsts.md]: Unless I' mistaken, the `always` should not be within the quotes.

**Comment by konklone** [comment]
This seems reasonable to me, though I haven't actually used it myself. For the benefit of those watching the thread, the nginx documentation says this about `always`:

> Adds the specified field to a response header provided that the response code equals 200, 201, 204, 206, 301, 302, 303, 304, or 307. A value can contain variables.
> 
> There could be several `add_header` directives. These directives are inherited from the previous level if and only if there are no `add_header` directives defined on the current level.
> 
> If the `always` parameter is specified (1.7.5), the header field will be added regardless of the response code.

**Comment by konklone** [comment]
Thanks, @inetbiz!

**Comment by inetbiz** [comment]
@konklone I hope you all send out a memo to various IT people to modify NGIX?

**Comment by konklone** [comment]
@inetbiz Unfortunately, there really isn't a good way to do that internally. It's just too big and decentralized of a space, without a lot of clear community connections. I wish there was a clear way to notify everyone known to use nginx in the federal government, about this and many other things!

**Comment by inetbiz** [comment]
Oh wow! Calling my FL Senator =)

**Comment by inetbiz** [comment]
Can someone get https://https.cio.gov/hsts/ modified? Their instructions do not include the 'always' param and they run NGIX.

**Comment by inetbiz** [comment]
@konklone who sets IT policy for all 3 branches of government? CyberCommand? I really do want to get Sen Marco Rubio involved to assist me. Just need to feed ammo to their staff so they liason correctly.

**Comment by konklone** [comment]
https://https.cio.gov/hsts/ is now updated. 

There is no IT policy body for all 3 branches of government. It's extremely decentralized. I'd also ask that we close discussion on that here, so as not to spam the 50+ folks who are watch-ing the repository for repo-specific topics.

**Pull request merged**